smartcontract/service/native/auth/auth.go

```diff
@@ -70,6 +71,9 @@ func InitContractAdmin(native *native.NativeService) ([]byte, error) {
 	}
 	invokeAddr := cxt.ContractAddress
 
+	if !ontid.VerifyID(param.AdminOntID) {
+		return utils.BYTE_FALSE, nil
```

Comment:
do we need provide some error information?

Reply:
In this case, the param provided is not valid, so I choose to return false for there is no internal-error happened. Maybe I should add some error-info log?